### ISSUE TITLE
Proselint not working on Atom 1.7.3

### ISSUE THREAD

**Issue by MaciejKlosowski** [AtomLinter/linter-proselint#23]
Hi all,

I have the following installed:

OS Win 10 
Python 3.5.1
pip 
proselint (via pip)
Atom 1.7.3
linter 1.11.4
linter-proselint 3.0.1

In Atom it says everytime that no issues were found, language is recognized as plain text. I tried to test this on different files (i.e. *.md, *.txt) but the result is still the same.

**Comment by Arcanemagus** [comment]
Does `proselint` on the CLI show any messages on the file?

**Comment by MaciejKlosowski** [comment]
Nope :/. It doesn't show any messages - I tried different combinations / texts (also from the website example) to trigger anything.

**Comment by Arcanemagus** [comment]
Since this is just a translator for what `proselint` shows, if it isn't showing anything then `linter-proselint` wouldn't be showing anything either.

**Comment by MaciejKlosowski** [comment]
I thought the whole point was to get error correction as per the example provided on demo page. Currently even if I write text with issues it doesn't perform any checks.

**Comment by Arcanemagus** [comment]
You said that `proselint` itself was finding nothing wrong with the file, if it isn't then either there is nothing wrong with the file, or there is something wrong with `proselint` that you should fix first before trying to use it within this package.

For example, with this file:

```markdown
It's 11 p.m. at night.
```

I get two messages from running `proselint` on the CLI:
![image](https://cloud.githubusercontent.com/assets/427137/15485416/f0909a42-20f3-11e6-8601-a80354ce5b0e.png)

And within Atom, through `linter-proselint` I see the same results:
![image](https://cloud.githubusercontent.com/assets/427137/15485430/11e338e4-20f4-11e6-8ca7-5718827813c3.png)

**Comment by MaciejKlosowski** [comment]
A few screens:

<img width="960" alt="linter_check" src="https://cloud.githubusercontent.com/assets/17146750/15485985/d04c257c-2142-11e6-9bf2-2d2cf73103c9.png">
<img width="960" alt="proselint_install_check" src="https://cloud.githubusercontent.com/assets/17146750/15485986/d071e1ea-2142-11e6-8d87-ff4432978c38.png">

I also tried to run proselint on specific file from CLI but nothing happens, i just get another blank line.
I have other linter packages installed (like Alex) which are working fine on my setup.

**Comment by Arcanemagus** [comment]
Like I said twice now, if `proselint` itself isn't showing anything, then that is an issue with `proselint` that you need to fix **before** attempting to use `linter-proselint`. `linter-proselint` depends on `proselint` working since it simply parses the output of that.

What do you get if you run `proselint '.\linter test2.md.txt'` on your above file?
![image](https://cloud.githubusercontent.com/assets/427137/15486282/6976f4ec-20f9-11e6-84ce-9abf06a71a03.png)

**Comment by MaciejKlosowski** [comment]
Result below. Btw. what version of python you currently running? Maybe I should downgrade from 3.5 to 2.7 and reinstall everything?

<img width="960" alt="proselint_test" src="https://cloud.githubusercontent.com/assets/17146750/15487430/9b404f06-214d-11e6-966e-ea4c8f5a26b6.png">

**Comment by Arcanemagus** [comment]
That's a possibility, I'm using Python 2.7.11, and when I try to use `proselint` through Python 3.5 I'm getting "failed to create process.". That _might_ be because the Python 3.5 `python` isn't in my PATH though.

**Comment by Arcanemagus** [comment]
Definitely doesn't look like `proselint` works with Python 3.5.1 at least, modified the PATH so that was the first found and I still can't run it.
